app/services/workflow_sequence_service.rb

```diff
@@ -0,0 +1,15 @@
+class WorkflowSequenceService
+  def list_by_sequence
+    ActsAsTenant.current_tenant.workflows
+  end
+
+  def update_sequences(workflow_ids)
```

Comment:
This `workflow_ids` should contain all existing workflow ids, right? Partial of workflow ids will mess the `sequence`. Suggest to rename to `update_all_sequences`, if this is the case.